src/Context/LoginContext.php

```diff
@@ -53,6 +54,26 @@ public function iAmLoggedInWithPermissions($permCode)
         $this->stepILogInWith($email, $password);
     }
 
+    /**
+     * Creates and login as a member of a group with the correct permissions.
+     * Example: Given I am logged in as a memeber of "ADMIN" group
+     *
+     * @Given /^I am logged in as a memeber of "([^"]*)" group$/
```

Comment:
```suggestion
     * Example: Given I am logged in as a member of "ADMIN" group
     *
     * @Given /^I am logged in as a member of "([^"]*)" group$/
```
Sorry, I should have noticed this typo last time.